deps/curl.mk

```diff
@@ -18,7 +18,14 @@ $(SRCDIR)/srccache/curl-$(CURL_VER)/configure: $(SRCDIR)/srccache/curl-$(CURL_VE
 $(BUILDDIR)/curl-$(CURL_VER)/config.status: $(SRCDIR)/srccache/curl-$(CURL_VER)/configure
 	mkdir -p $(dir $@)
 	cd $(dir $@) && \
-	$< $(CONFIGURE_COMMON) --includedir=$(build_includedir) --without-ssl --with-mbedtls=$(build_prefix) CFLAGS="$(CFLAGS) $(CURL_CFLAGS)" LDFLAGS="$(LDFLAGS) $(CURL_LDFLAGS)"
+	$< $(CONFIGURE_COMMON) --includedir=$(build_includedir) \
+	--disable-silent-rules --without-gnutls --without-gssapi \
```

Comment:
we don't want `disable-silent-rules` (it just makes `make` noisy, by defaulting to V=1 instead of V=0)